.gitignore

```diff
@@ -114,3 +114,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# jetbrains
+.idea/workspace.xml
```

Comment:
Takto ignoruješ iba jeden subór ale kopec ďalších sa pridalo do repa. Je potrebné to vyriešiť.
```suggestion
.idea/*
```

Reply:
Tak isto prosím odstráň všetky súbory  z `.idea/` (stačí iba z commitov).